setup.md

```diff
@@ -92,33 +176,55 @@ may overflow your screen. Set the device scale under the **Window > Scale** menu
 
 ### Deploy to iOS devices
 
-To deploy your Flutter app to a physical iOS device, you’ll need some additional tools:
+To deploy your Flutter app to a physical iOS device, you’ll need some additional tools, an account, and a profile:
 
 1. Install [homebrew](http://brew.sh/).
-2. Open the terminal and run this command to install the tools for deploying Flutter apps to
+1. Open the terminal and run this command to install the tools for deploying Flutter apps to
 iOS devices.
 
- ```
-  $ brew install ideviceinstaller ios-deploy
-```
-
-3. Follow the steps defined by Apple to configure Xcode with your Apple Developer account and enable deploying with provisioning profiles. To learn how to set up your profile, see [Apple's official documentation](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingProfiles/MaintainingProfiles.html).
-4. Select your Development Team for the Flutter project.
-  1. Open the default Xcode workspace in your project under `ios/Runner.xcworkspace`. 
-  2. Select the `Runner` project in the left navigation panel.
-  3. In the Target Settings page, make sure your Development Team is selected under **General > Signing > Team**.
-5. Start your app by running `flutter run`.
+   ```
+   $ brew install ideviceinstaller ios-deploy
+   ```
+1. If you see the following error:
+    ```
+   You must brew link libtasn1 libplist libusb before ideviceinstaller can be installed
```

Comment:
It's here because the user can get into a loop and not get out, unless they know about `brew doctor`. (Ask me how I know...)
@lukechurch pointed out offline that this might have been corp-related.
Let's see if external users get into the loop...